app/connector/webhook/pom.xml

```diff
@@ -50,6 +50,10 @@
       <groupId>io.syndesis.connector</groupId>
       <artifactId>connector-support-processor</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.syndesis.connector</groupId>
+      <artifactId>connector-support-processor</artifactId>
```

Comment:
This is declared twice now, Maven shows a warning:
```
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for io.syndesis.connector:connector-webhook:jar:1.9-SNAPSHOT
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: io.syndesis.connector:connector-support-processor:jar -> duplicate declaration of version (?) @ io.syndesis.connector:connector-webhook:[unknown-version], /workspace/app/connector/webhook/pom.xml, line 53, column 17
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
```